cmd/kyma/alpha/deploy/opts.go

```diff
@@ -98,14 +118,24 @@ func (o *Options) validateFlags() error {
 	if o.Domain != LocalKymaDevDomain && !o.tlsCertAndKeyProvided() {
 		return fmt.Errorf("To use a custom domain name also a custom TLS certificate and TLS key has to be provided")
 	}
-	if (o.TLSKey != "" || o.TLSCrt != "") && !o.tlsCertAndKeyProvided() {
+	if (o.TLSKeyFile != "" || o.TLSCrtFile != "") && !o.tlsCertAndKeyProvided() {
```

Comment:
There is also another check inside the function `tlsCertAndKeyProvided`. So, this if statement is confusing.

Reply:
Agree, new version is more expressive :)